apps/entity-service/src/entities/project-pitch.service.ts

```diff
@@ -0,0 +1,62 @@
+import { BadRequestException, Injectable, NotFoundException } from "@nestjs/common";
+import { ProjectPitch } from "@terramatch-microservices/database/entities";
+import { Includeable, Op } from "sequelize";
+import { PaginatedQueryBuilder } from "@terramatch-microservices/database/util/paginated-query.builder";
+import { MAX_PAGE_SIZE } from "./entities.service";
+import { ProjectPitchQueryDto } from "./dto/project-pitch-query.dto";
+
+@Injectable()
+export class ProjectPitchService {
+  async getProjectPitch(uuid: string) {
+    const projectPitch = await ProjectPitch.findOne({ where: { uuid } });
+    if (!projectPitch) {
+      throw new NotFoundException("ProjectPitch not found");
+    }
+    return projectPitch;
+  }
+
+  async getProjectPitches(query: ProjectPitchQueryDto) {
+    const pageNumber = query.page ? query.page.number : 1;
+    const pageSize = query.page ? query.page.size : MAX_PAGE_SIZE;
```

Comment:
This construct doesn't handle the case of `query.page.size` or `query.page.number` being null, and doesn't do any validation on the resulting values. It's also using implicit null checks (`query.page ?` instead of `query.page != null ?`). I like how the entities service does it:
```
    const { size: pageSize = MAX_PAGE_SIZE, number: pageNumber = 1 } = page ?? {};
    if (pageSize > MAX_PAGE_SIZE || pageSize < 1) {
      throw new BadRequestException(`Invalid page size: ${pageSize}`);
    }
    if (pageNumber < 1) {
      throw new BadRequestException(`Invalid page number: ${pageNumber}`);
    }
```